networkx/algorithms/tests/test_cycles.py

```diff
@@ -212,3 +214,44 @@ def test_dag(self):
                       find_cycle, G, orientation='original')
         x = list(find_cycle(G, orientation='ignore'))
         assert_equal(x, [(0,1,FORWARD), (1,2,FORWARD), (0,2,REVERSE)])
+
+
+class TestMinimumCycles:
+
+    def setUp(self):
+        T = nx.Graph()
+        T.add_cycle([1, 2, 3, 4], weight=1)
+        T.add_edge(2, 4, weight=5)
+        self.diamond_graph = T
+
+    def basis_equal(self, a, b):
```

Comment:
This should be a module-level function, not an instance method. It should also be called `assert_basis_equal()` to match the other assertion functions.